google/provider.go

```diff
@@ -106,6 +106,7 @@ func Provider() terraform.ResourceProvider {
 			GeneratedComposerResourcesMap,
 			GeneratedResourceManagerResourcesMap,
 			map[string]*schema.Resource{
+				"google_attached_disk":                         resourceAttachedDisk(),
```

Comment:
Let's call it `google_compute_attached_disk` to align with all the other compute engine resources